src/store/modules/stores.js

```diff
@@ -39,7 +39,7 @@ export const getters = {
   all: state => state.idList.map(i => state.entries[i]).sort(sortByName),
   byActiveGroup: (state, getters, rootState, rootGetters) => getters.all.filter(e => e.group === rootGetters['groups/activeGroupId']),
   get: state => (id) => state.entries[id],
-  withLocation: (state, getters) => getters.all.filter(e => e.longitude && e.latitude),
+  withLocation: (state, getters) => getters.byActiveGroup.filter(e => e.longitude && e.latitude),
```

Comment:
Hmm, as comment above, I find this unclear that it filters by active group without saying so in the name.

I would be happy without the `withLocation` getters, and rely on the map to filter those out (which it has to do anyway...)

Reply:
Yep, will remove the withLocation getters and handle it in the groupMap component